Fix the crash in unit test

Way to reproduce the error:
run  `go test ./...` from the repo root dir to reproduce it.

```
$ go test ./... 
?       github.com/StarRocks/starrocks-kubernetes-operator/cmd  [no test files]
?       github.com/StarRocks/starrocks-kubernetes-operator/common       [no test files]
?       github.com/StarRocks/starrocks-kubernetes-operator/components/offline_job       [no test files]
?       github.com/StarRocks/starrocks-kubernetes-operator/components/register_container        [no test files]
ok      github.com/StarRocks/starrocks-kubernetes-operator/internal/fe  (cached)
I0131 08:29:18.092066 3563771 starrockscluster_controller.go:80] StarRocksClusterReconciler reconciler the update crd name starrockscluster-sample namespace default
I0131 08:29:18.092842 3563771 k8sutils.go:56] Creating resource service namespace default name fe-domain-search kind 
I0131 08:29:18.092858 3563771 k8sutils.go:56] Creating resource service namespace default name starrockscluster-sample-fe-service kind 
I0131 08:29:18.092912 3563771 fe_statefulset.go:68] FeController buildStatefulSetParams [{"metadata":{"name":"fe-storage","creationTimestamp":null},"spec":{"accessModes":["ReadWriteOnce"],"resources":{"requests":{"storage":"10Gi"}},"storageClassName":"shard-data"},"status":{}}]
I0131 08:29:18.093138 3563771 fe_pod.go:75] FeController buildPodTemplate vols [{"name":"fe-storage","persistentVolumeClaim":{"claimName":"fe-storage"}}]
I0131 08:29:18.093281 3563771 k8sutils.go:56] Creating resource service namespace default name starrockscluster-sample-fe kind 
I0131 08:29:18.093305 3563771 statefulset.go:128] the statefulset new hash value 1286400549 old have value 4246122645 oldGeneration 0 new Generation 0
I0131 08:29:18.093309 3563771 fe_controller.go:93] FeController Sync exist statefulset not equals to new statefuslet
I0131 08:29:18.093313 3563771 k8sutils.go:64] Updating resource service namespace default name starrockscluster-sample-fe kind l&TypeMeta{Kind:,APIVersion:,}
I0131 08:29:18.094360 3563771 cn_controller.go:51] CnController Sync the cn component is not needed namespace default starrocks cluster name starrockscluster-sample
I0131 08:29:18.094366 3563771 starrockscluster_controller.go:141] StarRocksClusterReconciler reconcile status.
I0131 08:29:18.094368 3563771 starrockscluster_controller.go:178] StarRocksClusterReconciler reconciler namespace default name starrockscluster-sample
I0131 08:29:18.094440 3563771 starrockscluster_controller.go:80] StarRocksClusterReconciler reconciler the update crd name starrockscluster-sample namespace default
I0131 08:29:18.094542 3563771 k8sutils.go:56] Creating resource service namespace default name fe-domain-search kind 
I0131 08:29:18.094555 3563771 k8sutils.go:56] Creating resource service namespace default name starrockscluster-sample-fe-service kind 
--- FAIL: TestStarRocksClusterReconciler_FeReconcileSuccess (0.00s)
panic: cannot parse '': quantities must match the regular expression '^([+-]?[0-9.]+)([eEinumkKMGTP]*[-+]?[0-9]*)$' [recovered]
        panic: cannot parse '': quantities must match the regular expression '^([+-]?[0-9.]+)([eEinumkKMGTP]*[-+]?[0-9]*)$'

goroutine 14 [running]:
testing.tRunner.func1.2({0x16736e0, 0xc0003ffae0})
        /snap/go/10030/src/testing/testing.go:1396 +0x24e
testing.tRunner.func1()
        /snap/go/10030/src/testing/testing.go:1399 +0x39f
panic({0x16736e0, 0xc0003ffae0})
        /snap/go/10030/src/runtime/panic.go:884 +0x212
k8s.io/apimachinery/pkg/api/resource.MustParse({0x0, 0x0})
        /home/d.liu/go/pkg/mod/k8s.io/apimachinery@v0.25.0/pkg/api/resource/quantity.go:139 +0x1ce
github.com/StarRocks/starrocks-kubernetes-operator/pkg/fe_controller.(*FeController).buildStatefulSetParams(_, _, _)
        /home/d.liu/git/dengliu/starrocks-kubernetes-operator/pkg/fe_controller/fe_statefulset.go:60 +0x8fc
github.com/StarRocks/starrocks-kubernetes-operator/pkg/fe_controller.(*FeController).Sync(0xc000a73920, {0x1b025e8, 0xc000044158}, 0xc000338160)
        /home/d.liu/git/dengliu/starrocks-kubernetes-operator/pkg/fe_controller/fe_controller.go:75 +0x7aa
github.com/StarRocks/starrocks-kubernetes-operator/pkg.(*StarRocksClusterReconciler).Reconcile(0xc000a73590, {0x1b025e8, 0xc000044158}, {{{0x188475d, 0x7}, {0x189baa7, 0x17}}})
        /home/d.liu/git/dengliu/starrocks-kubernetes-operator/pkg/starrockscluster_controller.go:119 +0x522
github.com/StarRocks/starrocks-kubernetes-operator/pkg.TestStarRocksClusterReconciler_FeReconcileSuccess(0x0?)
        /home/d.liu/git/dengliu/starrocks-kubernetes-operator/pkg/starrockscluster_controller_test.go:171 +0x9bc
testing.tRunner(0xc000583380, 0x196d180)
        /snap/go/10030/src/testing/testing.go:1446 +0x10b
created by testing.(*T).Run
        /snap/go/10030/src/testing/testing.go:1493 +0x35f
FAIL    github.com/StarRocks/starrocks-kubernetes-operator/pkg  0.019s
?       github.com/StarRocks/starrocks-kubernetes-operator/pkg/apis     [no test files]
?       github.com/StarRocks/starrocks-kubernetes-operator/pkg/apis/starrocks/v1alpha1  [no test files]
?       github.com/StarRocks/starrocks-kubernetes-operator/pkg/be_controller    [no test files]
?       github.com/StarRocks/starrocks-kubernetes-operator/pkg/cn_controller    [no test files]
?       github.com/StarRocks/starrocks-kubernetes-operator/pkg/common   [no test files]
?       github.com/StarRocks/starrocks-kubernetes-operator/pkg/common/hash      [no test files]
ok      github.com/StarRocks/starrocks-kubernetes-operator/pkg/common/resource_utils    (cached)
ok      github.com/StarRocks/starrocks-kubernetes-operator/pkg/fe_controller    (cached)
?       github.com/StarRocks/starrocks-kubernetes-operator/pkg/k8sutils [no test files]
?       github.com/StarRocks/starrocks-kubernetes-operator/scripts/docs/template        [no test files]
FAIL

```

--- a/pkg/starrockscluster_controller_test.go
+++ b/pkg/starrockscluster_controller_test.go
@@ -98,6 +98,7 @@ func TestStarRocksClusterReconciler_FeReconcileSuccess(t *testing.T) {
 					{
 						Name:             "fe-storage",
 						StorageClassName: rutils.GetStringPointer("shard-data"),
+						StorageSize:      "10Gi",
 						MountPath:        "/data/fe/meta",
 					},
 				},
@@ -184,7 +185,8 @@ func TestStarRocksClusterReconciler_CnResourceCreate(t *testing.T) {
 			Namespace: "default",
 		},
 		Spec: v1alpha12.StarRocksClusterSpec{
-			ServiceAccount: "starrocksAccount",
+			ServiceAccount:  "starrocksAccount",
+			StarRocksFeSpec: &v1alpha12.StarRocksFeSpec{},
 			StarRocksCnSpec: &v1alpha12.StarRocksCnSpec{
 				Replicas: rutils.GetInt32Pointer(3),
 				Image:    "starrocks.com/cn:2.40",
@@ -222,7 +224,8 @@ func TestStarRocksClusterReconciler_CnStatus(t *testing.T) {
 			Namespace: "default",
 		},
 		Spec: v1alpha12.StarRocksClusterSpec{
-			ServiceAccount: "starrocksAccount",
+			ServiceAccount:  "starrocksAccount",
+			StarRocksFeSpec: &v1alpha12.StarRocksFeSpec{},
 			StarRocksCnSpec: &v1alpha12.StarRocksCnSpec{
 				Replicas: rutils.GetInt32Pointer(3),
 				Image:    "starrocks.com/cn:2.40",

